transform/operations_test.go

```diff
@@ -449,6 +449,23 @@ func TestCurrentTime(t *testing.T) {
 	runOpTests(t, func() transformOperation { return &currentTime{} }, tests)
 }
 
+func TestRemoveHTML(t *testing.T) {
+	tests := []opTests{
+		{
+			description: "Working case one",
+			in:          "Japanese man calls for a trial by combat when talking to his wife's <br>mother in law<br> &",
+			want:        "Japanese man calls for a trial by combat when talking to his wife's mother in law &",
+		},
+		{
+			description: "Working case two",
+			in:          "<div><br>++Japanese <i>man</i> Collin D'Oliver Murasame <strong>calls for a trial by combat when talking</strong> to his wife<custom>",
+			want:        "++Japanese man Collin D'Oliver Murasame calls for a trial by combat when talking to his wife",
+		},
+	}
```

Comment:
There are cases over in https://github.com/GannettDigital/content-api-authoring/pull/262 that failed replacement and would make good unit tests here.